start-minecraftFinalSetup

```diff
@@ -141,16 +141,48 @@ if isTrue "${USE_AIKAR_FLAGS}"; then
   "
 fi
 
-if isTrue "${USE_LARGE_PAGES}"; then
+# Pre-allocate the memory so we don't waste cycles on alloc/dealloc.
+if isTrue "${USE_MPD_FLAGS}"; then
```

Comment:
I can rework this and target at the master branch.

How would you feel about defaulting to testing/using large pages and preallocating the JVM heap, and just let the THP variable go unused/deprecated? Seems like it's always a good thing with Minecraft Server and should default on when it can.

MPD are my initials, and I can limit those changes to the GC stuff.

I'm very cool with making this multiarch-15, it might be time to remove all but 8, 11, & 15, as those JVMs are EOL. If we use the tag I'm proposing here, then we'll create multiarch-16 when 16 comes out, and nuke multiarch-15? I've already asked OpenJDK to make a 'latest-jvm' tag.

Reply:
I'm sorry, too many acronyms. What's THP?

I'd prefer not to name the option after a person, except Aikar's flags was a special case since there was a published article where the research was thoroughly documented and became named as such. Forgive me if you have indeed published articles on your combination of JVM flags -- those kinds of things, including Aikar's on not on my usual radar.

Yes, your detection logic alleviates the need for USE_LARGE_PAGES, so it turns out I'm fine with removing/deprecating the check for that and just use the detection logic you're proposing.

Instead of `USE_MPD_FLAGS` can you break these up into logical groups of flags:
- PRE_TOUCH_HEAP, to align with the JVM option name. Claiming it is "pre-allocated" vs the behavior of setting initial heap size is going to be confusing to people
- USE_SHENANDOAH_GC, to enable the detection and activation logic you're proposing

Reply:
Can do!